x-pack/plugins/apm/common/environment_filter_values.ts

```diff
@@ -22,11 +22,13 @@ const environmentLabels: Record<string, string> = {
 };
 
 export const ENVIRONMENT_ALL = {
+  esFieldValue: undefined,
```

Comment:
@dgieselaar This (`esFieldValue`) is being passed to the DOM elements causing the following error:

![image](https://user-images.githubusercontent.com/209966/120205653-7eff2f00-c22a-11eb-9073-5cfd056fda90.png)